src/telliot_feeds/reporters/tips/__init__.py

```diff
@@ -1,16 +1,47 @@
+from typing import Optional
+
 from multicall.constants import MULTICALL2_ADDRESSES
 from multicall.constants import MULTICALL3_ADDRESSES
 from multicall.constants import Network
+from multicall.constants import NO_STATE_OVERRIDE
 
 from telliot_feeds.queries.query_catalog import query_catalog
 
 
 # add testnet support for multicall that aren't avaialable in the package
-Network.PulsechainTestnet = 941
-MULTICALL2_ADDRESSES[Network.PulsechainTestnet] = "0x959a437F1444DaDaC8aF997E71EAF0479c810267"
-Network.Chiado = 10200
-MULTICALL3_ADDRESSES[Network.Chiado] = "0x08e08170712c7751b45b38865B97A50855c8ab13"
+def add_multicall_support(
+    network: str,
+    network_id: int,
+    state_override: bool = True,
+    multicall2_address: Optional[str] = None,
+    multicall3_address: Optional[str] = None,
+) -> None:
+    """Add support for a network that doesn't have multicall support in the package"""
+    if not hasattr(Network, network):
```

Comment:
I'd add a log msg if `Network` does not have the attribute checked for on line 20